helpdesk/views/staff.py

```diff
@@ -562,10 +545,10 @@ def get_due_date_from_request_or_ticket(
 
 
 def get_and_set_ticket_status(
-    new_status: str,
+    new_status: int,
     ticket: Ticket,
     follow_up: FollowUp
-) -> typing.Tuple[str, str]:
+) -> typing.Tuple[str, int]:
```

Comment:
This looks correct since there is a cast in the calling method:
`new_status = int(request.POST.get('new_status', ticket.status))`

Reply:
good point. does that make the cast redundant and able to be removed?